### PR TITLE
Updates pitching page

### PR DESCRIPTION
**Pull request by clausmullie**
We've updated our pitching page to reflect our current roadmap and intentions. This doesn't yet include an elevator pitch for codebase stewardship for public code (for niche/OSPO audiences), but that's a logical addition.

### CODE REVIEW

**Comment by ElenaFdR** [comment]
Re collaborating with OSPOs, we've created [this publiccode.net page](https://publiccode.net/OSPO) (which is effectly our pitch), but never linked to it from anywhere at all.

File: activities/communication/pitching.md [hunk 1]
```diff
@@ -61,8 +61,8 @@ We have different perspectives on open source:
 
 We should reflect our [founding principles](https://about.publiccode.net/organization/mission.html#founding-principles) in our pitches.
 
-Avoid: "we think or we believe" [open source is better]
-Instead, describe what we do: "we help municipalities [share collaborative software].
+Avoid: "we think or we believe" [open source is better].
+Instead, describe what we do: "we help public organizations [share collaborative software].
```
Comment (Ainali) [inline activities/communication/pitching.md:65]: This is of course technically more correct, but I often like to mention municipalities as it is less abstract and is often easy for people to relate to.

Comment (ericherman) [reply activities/communication/pitching.md:65]: It the pitching guidance for printed text as well? I was thinking that it was more about verbal, but maybe that is wrong. In printed text, I'd lean  be more technically correct and in engaged conversation be more loose and prefer something most easy to absorb.

Comment (clausmullie) [reply activities/communication/pitching.md]: reverted

File: activities/communication/pitching.md [hunk 2]
```diff
@@ -157,3 +157,8 @@ These are the audiences we've spoken to about the Foundation for Public Code:
 * activists
 * philanthropists
 * the general public
+
+## Further reading
```
Comment (ElenaFdR) [inline activities/communication/pitching.md:161]: @clausmullie This is a nice addition.

**Review by Ainali — APPROVED** [review]

**Comment by clausmullie** [comment]
closes #1340

**Pull request merged**